spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ImportExportControllerHelperService.java

```diff
@@ -359,15 +359,23 @@ public JSONObject confirmLibrariesPoolsUpload(HttpSession session, JSONObject js
                 try {
                   LibraryQC lqc = new LibraryQCImpl();
                   lqc.setLibrary(library);
-                  lqc.setInsertSize(insertSize);
                   lqc.setResults(Double.valueOf(jsonArrayElement.getString(5)));
                   lqc.setQcCreator(user.getLoginName());
                   lqc.setQcDate(new Date());
                   lqc.setQcType(requestManager.getLibraryQcTypeByName("Qubit"));
 
+                  LibraryQC lis = new LibraryQCImpl();
+                  lis.setLibrary(library);
+                  lis.setResults((double) insertSize);
+                  lis.setQcCreator(user.getLoginName());
+                  lis.setQcDate(new Date());
+                  lis.setQcType(requestManager.getLibraryQcTypeByName("Insert Size"));
```

Comment:
After discussion, Heather and I decided the reasonable workflow is to put the imported insert size on the library itself and not write a QC record for it. If the import changes it, then it will be recorded in the changelog.